### PR TITLE
added cask for vrep by coppelia robotics

### PR DESCRIPTION
**Pull request by ahundt**

### CODE REVIEW

File: Casks/vrep.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  version '3.2.1'
+  sha256 '8bf388f38ee215a819a2062b6ee4d29e4607afedd44856b0dc76fe075000fad3'
+
+  url "http://coppeliarobotics.com/V-REP_PRO_EDU_V#{version.gsub('.', '_')}_Mac.zip"
+  name 'vrep'
+  homepage 'http://www.coppeliarobotics.com/index.html'
+  license :other
+
+  suite "V-REP_PRO_EDU_V#{version.gsub('.', '_')}_Mac"
+
```
Comment (scribblemaniac) [inline Casks/vrep.rb]: Please remove the trailing whitespace on this line.

**Comment by ahundt** [comment]
I'm not really sure what went wrong here. Could someone advise me?

**Comment by scribblemaniac** [comment]
@ahundt Seems like you figured it out. Also, it looks like the name should be V-REP based on the homepage. When you are done making changes, please try and [squash your commits](http://davidwalsh.name/squash-commits-git). Thanks!

File: Casks/vrep.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  caveats do
+    free_license 'http://www.coppeliarobotics.com/licensing.html'
```
Comment (vitorgalvao) [inline Casks/vrep.rb]: Not really the appropriate usage, as it does not apply to everybody. This is to be used on discontinued apps.

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It needed some fixes, so they were made in 06b44ecfc3ba44f4e60e9de03a714b390f26cb8e. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Comment by ahundt** [comment]
cool, thanks! sorry I've been out of town the last few days. I need to figure out how to do what you did with squash commits + modify + credit.

**Pull request closed without merging**